FlightReservation/components/myFlightInfo.js

```diff
@@ -0,0 +1,43 @@
+import React, {useState} from 'react';
+import {StyleSheet, View, Text, Image} from 'react-native';
+import colors from '../src/colors';
+import LocationComponent from './flightsComponents/locationComponent';
+import AirplaneIcon from './flightsComponents/airplaneIcon';
+import DateFlight from './flightsComponents/dateFlight';
+import NumPassengers from './flightsComponents/numPassengers';
+import TripInfo from '../components/tripInfo';
```

Comment:
imported components are not used in this file

Reply:
we delete that lines!